sig-scalability/throughput_burts_slo.md

```diff
@@ -0,0 +1,23 @@
+# SLO: Kubernetes cluster of size at least X is able to start Y Pods in Z minutes
+
```

Comment:
Can we mention that this is not something we are meeting, just something we are planning to have.